src/components/block-button.js

```diff
@@ -0,0 +1,31 @@
+window.blockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "block", whom: id });
+  NAF.connection.entities.removeEntitiesOfClient(id);
+};
+
+window.unblockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "unblock", whom: id });
+  window.setTimeout(() => {
+    // Wait enough time for the unblock to be processed
+    NAF.connection.entities.completeSync(id);
+  }, 1000);
+};
+
+AFRAME.registerComponent("block-button", {
+  init() {
+    this.el.addEventListener("click", this.onClick);
+    const owner = (function getOwner(el) {
```

Comment:
Use the NAF method to find the networked element for an entity like in https://github.com/networked-aframe/networked-aframe/blob/master/src/components/networked-audio-source.js#L22